src/common/commandExecutor.ts

```diff
@@ -84,11 +84,11 @@ export class CommandExecutor {
         });
 
         result.stderr.on("data", (data: Buffer) => {
-            Log.logMessage(data.toString(), /*formatMessage*/false);
+            Log.logStreamData(data, process.stderr);
         });
 
         result.stdout.on("data", (data: Buffer) => {
-            Log.logMessage(data.toString(), /*formatMessage*/false);
+            Log.logStreamData(data, process.stdout);
```

Comment:
I think I'd be better to have two messages: logStreamOutput and logStreamError so we don't need to pass the extra parameter around.
Still, I think this is a good enough fix for this release, and we can make it nicer later.
:+1: